### PR TITLE
- implemented Python wrappers for computing PMI axes and moments

### PR DESCRIPTION
**Pull request by ptosco**
#### Reference Issue
Request from Pavel Polischuk:
https://www.mail-archive.com/rdkit-discuss@lists.sourceforge.net/msg07600.html

#### What does this implement/fix? Explain your changes.
Implements Python wrappers for computePrincipalAxesAndMoments() and computePrincipalAxesAndMomentsFromGyrationMatrix()

### CODE REVIEW

**Comment by ptosco** [comment]
@greglandrum Hi Greg, if you wish to take a look I have implemented the Python wrappers for `computePrincipalAxesAndMoments()` and `computePrincipalAxesAndMomentsFromGyrationMatrix()`. Thanks!

File: Code/GraphMol/MolTransforms/Wrap/rdMolTransforms.cpp [hunk 1]
```diff
@@ -40,6 +40,68 @@ PyObject *computeCanonTrans(const Conformer &conf,
...
+    i = 0;
+    for (size_t y = 0; y < 3; ++y) {
+      for (size_t x = 0; x < 3; ++x)
+        axesNpyData[i++] = axes(y, x);
+    }
+    PyArrayObject *momentsNpy = (PyArrayObject *)PyArray_SimpleNew(1, dims, NPY_DOUBLE);
+    double *momentsNpyData = reinterpret_cast<double *>(PyArray_DATA(momentsNpy));
+    i = 0;
+    for (size_t y = 0; y < 3; ++y)
+      momentsNpyData[i++] = moments(y);
```
Comment (greglandrum) [inline Code/GraphMol/MolTransforms/Wrap/rdMolTransforms.cpp]: minor: you don't need `i` here.

**Comment by ptosco** [comment]
@greglandrum Good point, copy/paste overlook. Fixed.

**Comment by greglandrum** [comment]
Thanks. I'll merge as soon as travis is working again and the build goes through.

**Comment by greglandrum** [comment]
Thanks @ptosco

**Pull request merged**